doc/design/zarr-support.md

```diff
@@ -98,3 +98,38 @@ Honestly, not that unreasonable for this much data.
 Because the checksum has a well-defined ordering, you cannot decide that you want to tack a few more files on at the beginning after you have begun the checksumming.
 * Possibly inefficient hashing algorithm?
 My approach was simple but possibly naive, maybe there's a better algorithm that meets the requirements.
+
+## Benchmarks
+I mocked up API endpoints that would behave more or less like the ones described above.
+I recommend throwing the code away, but it should give a good estimate for performance.
```

Comment:
I strongly suggest *not* throwing the code away, but instead transferring it to a gist so we don't have to keep a weird branch around in the codebase.